README.md

```diff
@@ -43,3 +43,46 @@ A general purpose class that prompts a user for information. See [inquirer NPM M
 ## Flags
 
 Flags is a convenience reference to [@oclif/core#Flags](https://github.com/oclif/core/blob/main/src/flags.ts)
+
+### Specialty Flags
+
+These flags can be imported into a command and used like any other flag. See code examples in the links
+
+- [apiVersion](src/flags/apiVersion.ts)
+  - specifies a Salesforce API version.
+  - reads from Config (if available)
+  - validates version is still active
+  - warns if version if deprecated
+- [dirExsts](src/flags/fsFlags.ts)
+  - validates that directory specified exists and is a directory
+- [fileExsts](src/flags/fsFlags.ts)
+  - validates that file specified exists and is a file
+- [requiredOrgFlag](src/flags/orgFlags.ts)
+  - accepts a username or alias
+  - aware of configuration defaults
+  - throws if org or default doesn't exist or can't be found
+- [optionalOrgFlag](src/flags/orgFlags.ts)
+  - accepts a username or alias
+  - aware of configuration defaults
+  - might be undefined if an org isn't found
+- [requiredHubFlag](src/flags/orgFlags.ts)
+  - accepts a username or alias
+  - aware of configuration defaults
+  - throws if org or default doesn't exist or can't be found
+  - throws if an org is found but is not a dev hub
+
+### Flag Builders
+
+These functions accept parameters and return a flag with configured behavior. See code examples in the link
```

Comment:
Why not make all the above flags configurable like these?

Reply:
I thought about it.  I couldn't decide which was worse...having to use the parameterized flavor when not necessary, or having 2 flavors.

Reply:
done!